index.html

```diff
@@ -949,10 +957,20 @@ <h3>
         <ol>
           <li>Let |promise:Promise| be [=a new promise=].
           </li>
-          <li>If [=this=]'s [=relevant global object=]'s [=associated
-          Document=] is not [=allowed to use=] the [=policy-controlled
-          feature=] named `"hid"`, [=reject=] |promise| with a
-          "{{SecurityError}}" {{DOMException}} and return |promise|.
+          <li>Let |document:Document| be null</li>
```

Comment:
Use backticks for `null` and add a period. Each item should be a complete sentence.

    <li>Let |document:Document| be `null`.</li>

Reply:
Done.